src/js/components/Beacon/BeaconQueryUi.tsx

```diff
@@ -122,17 +189,47 @@ const BeaconQueryUi = () => {
     setFilters([STARTER_FILTER]);
     form.resetFields();
     form.setFieldsValue(formInitialValues);
+    clearFormError();
+    setErrorAlertClosed(false);
     launchEmptyQuery();
   };
 
-  const SearchToolTip = ({ text }: { text: string }) => {
+  const handleValuesChange = (_: Partial<FormValues>, allValues: FormValues) => {
+    form.validateFields(['Chromosome', 'Variant start', 'Variant end', 'Reference base(s)', 'Alternate base(s)']);
```

Comment:
this list of fields is used once above too for `const empty` - maybe worth factoring out?

Reply:
(all fields, i guess?)

Reply:
yeah, there are a lot of variant form names lying around in the code (and not too helpfully the correct name for this collection is something like allVariantFormFieldsExceptAssemblyID... )

In `empty` we're checking each name individually, where in validate fields we're using all names together. So a possible refactor is something like:

`const empty  = !allVariantFormFieldsExceptAssemblyID.some(field => formValues[field])`

but i'm not sure this is better....

Reply:
could use the every() function of the array (plus a boolean cast of !! for formValues[field])

in this case, why is assembly ID not validated?

Reply:
The reasoning behind the variants form is that if you want to search variants you fill it in, and if you don't, you leave it blank. So the if the form is blank it's ignored, and none of the rules about required fields are applied. 

However, the form is almost never blank, since the assembly ID field is already filled for you in the (very common) case where there is only one assembly to search from. That's why it's treated separately from everything else.